uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/tabs/absence.js

```diff
@@ -0,0 +1,26 @@
+var Promise = require('es6-promise').Promise;
+var tab = require('./tab');
+
+module.exports = (function () {
+  return tab.extend({
+    readySelector: '.absence-tab-page',
+    tabTitle: 'Absence',
+
+    /**
+     * Opens subtab of the Absence tab
+     *
+     * @param  {string} mode "report", "calendar", "entitlements" or "workpatterns"
+     * @return {Promise}
+     */
+    openSubTab: function (mode) {
```

Comment:
I dont think this function/file is required. I have implemented the same without adding this, here is the code
https://github.com/civicrm/civihr/pull/1964/files#diff-acc78549baf2880d6cb40bed64b15046R10

Your architecture and mine for backstop js is not same(not saying yours is wrong), but both of us should follow the same folder/code structure to keep consistency, if you have any suggestion to the above code I provided, let discuss that to come to an agreement which we both can follow.